devenv/dev-dashboards/datasource-loki/loki_fakedata.json

```diff
@@ -0,0 +1,421 @@
+    },
+    {
+      "datasource": {
+        "type": "loki",
+        "uid": "PDDA8E780A17E7EF1"
+      },
+      "fieldConfig": {
+        "defaults": {
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "red",
+                "value": 80
+              }
+            ]
+          },
+          "unit": "short"
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 7,
+        "w": 12,
+        "x": 0,
+        "y": 18
+      },
+      "id": 20,
+      "options": {
+        "colorMode": "value",
+        "graphMode": "area",
+        "justifyMode": "auto",
+        "orientation": "auto",
+        "reduceOptions": {
+          "calcs": [
+            "lastNotNull"
+          ],
+          "fields": "",
+          "values": false
+        },
+        "textMode": "auto"
+      },
+      "pluginVersion": "9.3.0-pre",
+      "targets": [
+        {
+          "datasource": {
+            "type": "loki",
+            "uid": "PDDA8E780A17E7EF1"
+          },
+          "editorMode": "code",
+          "expr": "count_over_time({place=\"moon\"} [5m])",
+          "legendFormat": "{{place}}",
+          "queryType": "range",
+          "refId": "A"
+        }
+      ],
+      "title": "Moon Log count over 5m",
+      "type": "stat"
+    },
+    {
+      "datasource": {
+        "type": "loki",
+        "uid": "PDDA8E780A17E7EF1"
+      },
+      "fieldConfig": {
+        "defaults": {
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "red",
+                "value": 80
+              }
+            ]
+          },
+          "unit": "short"
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 7,
+        "w": 12,
+        "x": 12,
+        "y": 18
+      },
+      "id": 19,
+      "options": {
+        "colorMode": "value",
+        "graphMode": "area",
+        "justifyMode": "auto",
+        "orientation": "auto",
+        "reduceOptions": {
+          "calcs": [
+            "lastNotNull"
+          ],
+          "fields": "",
+          "values": false
+        },
+        "textMode": "auto"
+      },
+      "pluginVersion": "9.3.0-pre",
+      "targets": [
+        {
+          "datasource": {
+            "type": "loki",
+            "uid": "PDDA8E780A17E7EF1"
+          },
+          "editorMode": "code",
+          "expr": "count_over_time({place=\"luna\"} [5m])",
+          "legendFormat": "{{place}}",
+          "queryType": "range",
+          "refId": "A"
+        }
+      ],
+      "title": "Luna Log count over 5m",
+      "type": "stat"
+    }
+  ],
+  "refresh": "5s",
```

Comment:
This is too often which results (in my case at least) in `too many outstanding requests`. I'd suggest this to be either `off`, or something like `1m`